musicdao/src/main/java/com/example/musicdao/wallet/WalletService.kt

```diff
@@ -32,6 +34,16 @@ class WalletService(val walletDir: File, private val musicService: MusicService)
     private var started = false
     var percentageSynced = 0
 
+    private var currentTransactionAmount: Long = 0
```

Comment:
we can still improve this code a lot, is it really needed in the master right now?